libs/components/flyout/src/lib/modules/flyout/types/flyout-config-internal.ts

```diff
@@ -0,0 +1,30 @@
+import { SkyFlyoutConfig } from './flyout-config';
+
+/**
+ * Type to represent the flyout configuration after default values have been applied.
+ * @internal
+ */
+export interface SkyFlyoutConfigInternal extends SkyFlyoutConfig {
```

Comment:
I couldn't find any uses of this interface in this PR. Is it needed?

Reply:
It is used within `SkyFlyoutComponent` to account for where we have a config with optional properties but then set them in the `attach` function so that we can account that the properties will be set later. Happy to chat if you don't like what I did there.